Packs/SentinelOne/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "SentinelOne",
     "description": "Endpoint protection",
     "support": "partner",
-    "currentVersion": "3.2.9",
+    "currentVersion": "3.3.0",
```

```suggestion
    "currentVersion": "3.2.10",
```
its nothing major we are just fixing a bug, should not bump the version this much